SKQueue.swift

```diff
@@ -165,27 +165,27 @@ class SKQueue {
         while (keepWatcherThreadRunning) {
             let n = kevent(fd, nil, 0, &ev, 1, &timeout)
             if n > 0 && ev.filter == Int16(EVFILT_VNODE) && ev.fflags != 0 {
-                let pathEntry = Unmanaged<SKQueuePath>.fromOpaque(COpaquePointer(ev.udata)).takeUnretainedValue()
+                let pathEntry = Unmanaged<SKQueuePath>.fromOpaque(UnsafeRawPointer(ev.udata)).takeUnretainedValue()
                 let notification = SKQueueNotification(rawValue: ev.fflags)
-                dispatch_async(dispatch_get_main_queue()) {
-                    self.delegate?.receivedNotification(self, notification, forPath: pathEntry.path)
+                DispatchQueue.main.async {
+                  self.delegate?.receivedNotification(queue: self, notification, forPath: pathEntry.path)
                 }
             }
         }
         
         if close(fd) == -1 {
-            NSLog("SKQueue watcherThread: Couldn't close main kqueue (%d)", errno)
+            print("SKQueue watcherThread: Couldn't close main kqueue (\(errno))")
```

Comment:
`NSLog` is not interchangeable with `print` (NSLog writes to the system console). This should be left as-is.